pages/windows/get-childitem.md

```diff
@@ -1,6 +1,7 @@
 # Get-ChildItem
 
 > List items in a directory.
+> This command can only be used through `powershell`.
```

Comment:
I think we are talking about the program itself, :thinking:  and it's spelled in this way
```suggestion
> This command can only be used through PowerShell.
```